app/models/article.rb

```diff
@@ -211,6 +204,18 @@ class Article < ApplicationRecord
     boosted_additional_tags String, default: ""
   end
 
+  def self.active_help
+    stories = published
+      .cached_tagged_with("help")
+      .order(created_at: :desc)
+      .where("published_at > ? AND comments_count < ? AND score > ?", 12.hours.ago, 6, -4)
+    return stories if stories.size.positive?
```

Comment:
Would this maybe be easier?

```suggestion
    return stories if stories.present?
```

That said, the two alternatives seems to be almost the same, so maybe something like the following would also work:

```ruby
stories = published.cached_tagged_with("help").order(created_at: :desc)

stories.where("published_at > ? AND comments_count < ? AND score > ?", 12.hours.ago, 6, -4).presence ||
  stories
```

Reply:
This is an excellent suggestion! I've made this change and have added it all to the original `scope` rather than in the class method. 😊